### DISCUSSION TITLE
Seeking Swedish person regarding tip on config

### DISCUSSION THREAD

**Discussion by digitalit** [blakeblackshear/frigate#2474] [Ask A Question]
Hi guys,
For starters, i'm really sorry that my skills (not yet) allow me to create a correct issue with all required log files and what not.

I just wonder if there are any Swedish people that can help me with some questions.

Have a nice weekend 

Best regards
/Peter Carlsson

**Comment by parhedberg** [comment]
Jag kanske kan assistera dig. Vad är frågan?

Den lör 18 dec. 2021 10:29Peter Carlsson ***@***.***> skrev:

**Reply by digitalit** [reply]
Tack snälla för ditt snabba svar @parhedberg ,

Jag kör en hyfsat kraftig server med en mjukvara som heter unRaid. Den tillåter mig att köra både Docker appar och VM:s

Kan nämna också att min VM körs i en Raidad cache SSD så prestandan är finfin.

Jag har en VM enbart för Hassos (eller vad det korrekta namnet är.) Den har jag tilldelat 4096MB RAM och två CPU:er.

Det fungerar kanonbra och inga som helst prestandaproblem. Sedan kör jag Frigate NVR version: 2.3 med stor framgång. 

Jag har en extern USB GoogleCoral vilket alla rekommenderade i stället för att belasta CPU:s med recognitions.

Jag har 4st D-Link DCS-4602EV som jag är jättenöjda med men denna configurationsfil ger mig gråa hårstrån dvs. CONFIG/frigate.yml

Finns det någon möjlighet att jag får visa den för dig och om du kanske skulle kunna förkasta eller godkänna den?

Vill förtydliga att jag inte på något sätt är ute att få "kodhjälp" dvs. att någon gör något jobb åt mig. Endast en eventuell fingervisning om hur jag kanske bör tänka när jag eventuellt börjar koda om vissa partier.

Med vänlig hälsning,
/Peter

**Reply by parhedberg** [reply]
Det låter rätt likt min uppsättning, men jag kör proxmox och inte unraid. Annars lika.

Se om du kan skicka mig ett pm här på github?

**Reply by digitalit** [reply]
Hittar inget sätt att skicka något privat till dig. Har du någon annan idé?

Vore skönt att slippa "städa hela filen"

Jobbar även som brandman och blev nu kallad så jag återkommer så fort jag kan.

Tack

**Comment by parhedberg** [comment]
Jag ska också iväg, men har lite mer tid imorgon eftermiddag. Vet inte om
jag vill skriva mailadressen här..

Den lör 18 dec. 2021 14:15Peter Carlsson ***@***.***> skrev:
